### DISCUSSION TITLE
Why was the firmware directory deleted form the repo

### DISCUSSION THREAD

**Discussion by benstieg** [jomjol/AI-on-the-edge-device#1039] [General] [answered]
Hi @jomjol, 

I had the camera lying around for a while and wanted to start to build my smart water meter today. But I noticed, that you deleted the firmware directory from the repo in your latetes commit. Is there any specific reason why you did that? 

Best Regards,
Ben

**Comment by caco3** [comment, marked as answer]
The firmware is now provided through the release, see https://github.com/jomjol/AI-on-the-edge-device/releases

If you want to live on the edge and use experimental versions, you can download them from https://github.com/jomjol/AI-on-the-edge-device/actions?query=branch%3Arolling. Be aware hat they might be unstable and need manual reflashing through USB!

**Reply by benstieg** [reply]
Sorry, I was mislead by an not yet updated link in the wiki, which pointed to the no more existing firmware folder in the sources (see https://github.com/jomjol/AI-on-the-edge-device/wiki/Installation#firmware-flashing) 
I did fix the link to point to the releases.

**Reply by caco3** [reply]
great, thanks!
